Core/Core/Extensions/UIImageInstIconsExtensions.swift

```diff
@@ -27,6 +27,8 @@ extension UIImage {
     }
 
     public enum InstIconName: String, CaseIterable {
+        static let page = InstIconName.document
```

Comment:
this will get blown away the next time the script runs. Perhaps we should add some aliases to the script to generate these?

Reply:
Oh duh! Yeah I'll play with aliases. Good catch!

Reply:
I think aliases would be cool but it's easier to just use document for now 😆